docs/data/hubble/developer-guide/backfill/README.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: "Backfill"
+sidebar_position: 10
+---
+
+import DocCardList from "@theme/DocCardList";
+
+Once your scheduling and orchestration are set up, you might encounter the following scenarios:
+
+- **Bug Fix:** You resolve a bug and need to re-ingest a specific data column.
+- **New Feature:** You add a new data column as part of a feature request and need to backfill data for the newly added column.
+- **Raw Data Extraction:** You want to use Hubble as a source for raw data (XDR columns) and extract only the required data columns. For scenarios 1 and 2, you can perform a backfill using Airflow and trigger a Directed Acyclic Graph (DAG) for past dates.
```

Comment:
I like how you've outlined the different scenarios for when you need to perform a backfill. I think that's very clear. I think you should have a similar section that outlines the different options for how to backfill ie. 
Options:
1. Run stellar-etl and trigger a DAG for past dates
2. Export data from Bigquery (via SDK connection + SQL or exporting the data into files) 
3. JS UDF

This would allow you to link out to subpages that give more detail as necessary on these options

Reply:
Addressed in https://github.com/stellar/stellar-docs/pull/1362/commits/625b3bca8aa0fbd8a896a748558057e61ae863d3